### PR TITLE
Issue 83667

### PR DESCRIPTION
**Pull request by careri**
Fixes bug:
https://github.com/dotnet/runtime/issues/83667

Also fixed problem with opening the .devcontainer on Windows.

### CODE REVIEW

**Comment by careri** [comment]
@dotnet-policy-service agree

File: .devcontainer/scripts/onCreateCommand.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "Changing owner of workspace to vscode"
```
Comment (eiriktsarpalis) [inline .devcontainer/scripts/onCreateCommand.sh:3]: I would recommend keeping this change out. Feel free to file a second a PR if this is something that needs addressing.

**Review by eiriktsarpalis — COMMENTED** [review]
Thank you for your contribution!

File: .devcontainer/scripts/onCreateCommand.sh (hunk 1 above)
Comment (careri) [reply .devcontainer/scripts/onCreateCommand.sh]: You´re right I reverted it. My bad

**Review by eiriktsarpalis — APPROVED** [review]

**Pull request merged**